app/src/main/java/com/findmyrecycling/MainViewModel.kt

```diff
@@ -14,7 +14,7 @@ class MainViewModel(var productService : ProductService = ProductService()) : Vi
 
     fun fetchProducts() {
         viewModelScope.launch {
-            var innerProducts = productService.fetchProducts()
```

Comment:
A good suggestion, but it was covered in a previous review.  When that occurs, first in wins.